rust/agama-lib/src/security/client.rs

```diff
@@ -44,7 +46,7 @@ impl SecurityClient<'_> {
             .map(|(alg, value)| {
                 Ok(SSLFingerprint {
                     fingerprint: value,
-                    algorithm: alg.try_into()?,
+                    algorithm: super::model::SSLFingerprintAlgorithm::from_str(&alg)?,
```

Comment:
I would prefer to import the `SSLFingerprintAlgorithm` as you did with `SSLFingerprint` and `SecuritryProxy`.